wiki/markdown/running-eiffel-intelligence.md

```diff
@@ -26,23 +29,64 @@ With properties added to the maven command, e.g:
+ ## Running with Java command 
 
-    java -jar eiffel-intelligence-<version>.war
+Another option is to run the executable war file with java command.
+If running from source code, war file is generated and produced by maven command (mvn install command can be used as well):
+
+    mvn package -DskipTests
+
+ This command should produce an `eiffel-intelligence-<version>.war` file in target folder, `target/eiffel-intelligence-<version>.war`. 
 
-if you want to run with default configuration for local message buss and
-Mongo DB.
+The war file is executed by the following command and with default configuration:
+
+    java -jar eiffel-intelligence-<version>.war
 
 Own configuration can be provided with
 
-    java -jar eiffel-intelligence-<version>.war --spring.config.location=file:<path to own application.properties>
+    java -jar eiffel-intelligence-<version>.war --spring.config.location=<path to own application.properties>
 
 remember to keep the name of the properties file if you are a beginner to
 Spring. More advanced Spring user can look [here](https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-external-config.html)
 for more information about using external configuration.
 
-If only few properties need to be overridden then use java opts, for example
+If a few properties need to be overridden, then use java opts, for example:
 
     java -jar eiffel-intelligence-<version>.war -Dspring.data.mongodb.port=27019
 
-All available Eiffel-Intelligence properties can be found in [application.properties](https://github.com/Ericsson/eiffel-intelligence/blob/master/src/main/resources/application.properties) example file.
\ No newline at end of file
+## Running in Tomcat instance
+
+To run Eiffel Intelligence in Tomcat, the war file must be put into the webapp folder in tomcat installation folder, also called catalina home folder:
+
+    (catalina home)/webapp/
+
+If Eiffel Intelligence should be run without any context-path in the url address, then overwrite ROOT.war file in webapp folder with `eiffel-intelligence-<version>.war`:
+
+    cp eiffel-intelligence-<version>.war (catalina home)/webapp/ROOT.war
+
+Remove "ROOT" folder in webapp folder:
+
+    rm -rf (catalina home)/webapp/ROOT/
+
+Create "config" folder in webapp folder, if it doesn't exists. Spring and Eiffel Intelligence will look for the application.properties configuration file in config folder:
+
+    mkdir (catalina home)/webapp/config
+
+Copy the application.properties file into the newly created config folder:
+    
+    cp application.properties (catalina home)/webapp/config
+
+Start Tomcat and Eiffel Intelligence in background/daemon mode by executing command:
+
+    (catalina home)/bin/catalina.sh start
+
+To run Tomcat and Eiffel Intelligence with logs printed to console:
+    
+    (catalina home)/bin/catalina.sh run
+
+## Eiffel-Intelligence configurations and properties
```

Comment:
Missed one here. Replace the hyphen with space.

Reply:
fixed.